kinder/pkg/test/workflow/taskCmdBuilder.go

```diff
@@ -85,7 +85,7 @@ func newTaskCmdBuilder(w *Workflow) (c *taskCmdBuilder, err error) {
 
 // defines a list of custom utility functions that can be used in workflow templates
 var funcMap = template.FuncMap{
-	"resolve": kextract.ResolveLabel, // e.g. used in templates >> stable: '{{ resolve "release/stable" }}' or {{ "ci/latest" | resolve }}
+	"resolve": kextract.ResolveLabel, // e.g. used in templates >> stable: '{{ resolve "release/latest-1.14" }}' or {{ "ci/latest" | resolve }}
```

Comment:
should continue to be ` "release/stable"` it's just an example in the comment.

Reply:
done